index.html

```diff
@@ -15,68 +16,87 @@ <h2 class="title">INTENTION TIMER</h2>
+          <div id="buttonBox" class="button-box">
+            <button id="study" class="study">
+              <img id="studyActive" class="study-active hidden" src="./assets/study-active.svg" alt="Study Selected">
+              <img id="studyImage" class="study-image"src="./assets/study.svg" alt="">Study
+            </button>
+            <button id="meditate" class="meditate">
+              <img id="meditateActive" class="meditate-active hidden" src="./assets/meditate-active.svg" alt="Meditate Selected">
+              <img id="meditateImage" class="meditate-image" src="./assets/meditate.svg" alt="">Meditate
+            </button>
+            <button id="exercise" class="exercise">
+              <img id="exerciseActive" class="exercise-active hidden" src="./assets/exercise-active.svg" alt="Exercise Selected">
+              <img id="exerciseImage" class="exercise-image" src="./assets/exercise.svg" alt="">Exercise
+            </button>
           </div>
           <section class="error invisibility" id="buttonError">
-            <img class="error-image" src="./assets/warning.svg">
+            <img class="error-image" src="./assets/warning.svg" alt="Warning Error Select Activity Icon">
             <h4>Please select a category.</h4>
           </section>
-          <label class="accomplish">What would you like to accomplish during this time?</label>
-          <input id="accomplish" class="user-accomplish" type="text">
+          <label class="accomplish">What would you like to accomplish during this time?
+          </label>
+          <input id="accomplish" class="user-accomplish" type="text" alt="Enter Activity Description">
           <section class="error invisibility" id="accomplishError">
-            <img class="error-image" src="./assets/warning.svg">
+            <img class="error-image" src="./assets/warning.svg" alt="Warning Error Enter Text">
             <h4>A description is required.</h4>
           </section>
-          <div class="time-box">
-            <div class="minute-box">
+          <section class="time-box">
+            <section class="minute-box">
               <label>Minutes</label>
-              <input id="minutes" class="user-minutes" type="text">
+              <input id="minutes" class="user-minutes" type="text" alt="Enter Activity Minutes">
               <section class="error invisibility" id="minutesError">
-                <img class="error-image" src="./assets/warning.svg">
+                <img class="error-image" src="./assets/warning.svg" alt="Warning Error Enter Minutes Value">
                 <h4>Please input a numerical value.</h4>
               </section>
-            </div>
-            <div class="seconds-box">
+            </section>
+            <section class="seconds-box">
               <label>Seconds</label>
-              <input id="seconds" class="user-seconds" type="text">
+              <input id="seconds" class="user-seconds" type="text" alt="Enter Activity Seconds Here">
```

Comment:
Consider as an enhancement adding a placeholder value to the inputs to help the user use the correct format

Reply:
Great idea! Done.